domes-toolscript.gd

```diff
@@ -0,0 +1,7 @@
+@tool
```

Comment:
this is a "tool script", check it out: https://docs.godotengine.org/en/stable/tutorials/plugins/running_code_in_the_editor.html#how-to-use-tool

this only runs in the editor, so I could see the dome names while coding the connections